eng/pipelines/runtime.yml

```
@@ -105,14 +105,18 @@ extends:
       - template: /eng/pipelines/common/platform-matrix.yml
         parameters:
           jobTemplate: /eng/pipelines/common/global-build-job.yml
-          buildConfig: ${{ variables.debugOnPrReleaseOnRolling }}
+          buildConfig: Release
           platforms:
           - browser_wasm
           - browser_wasm_win
           jobParameters:
             nameSuffix: AllSubsets_CoreCLR
-            buildArgs: -s clr.runtime+libs -rc Release -c Release -lc $(_BuildConfig)
+            buildArgs: -s clr+libs -rc Release -c Release -lc Release
             timeoutInMinutes: 120
+            postBuildSteps:
+              - template: /eng/pipelines/common/templates/runtimes/build-runtime-tests.yml
+                parameters:
+                  testBuildArgs: test JIT/interpreter/Interpreter.csproj /p:LibrariesConfiguration=Release
```

```suggestion
            buildArgs: -s clr+libs -c $(_BuildConfig)
timeoutInMinutes: 120
            postBuildSteps:
              - template: /eng/pipelines/common/templates/runtimes/build-runtime-tests.yml
                parameters:
                  testBuildArgs: test JIT/interpreter/Interpreter.csproj
```
Or am I missing something?

---

`src/tests` needs the build to be Release. At the same time release coreclr interp doesn't work yet. But we are not trying to run it yet on this PR, just to build it.